projects/BFDR.Tests/FetalDeathRecord_Should.cs

```diff
@@ -896,8 +896,56 @@ public void TestSetDeliveryLocation()
       Assert.Equal("Bob's Medical Center", br.BirthFacilityName);
       Assert.Equal("Abignale Hospital", br.FacilityMotherTransferredFrom);
     }
+    
+    [Fact]
+    public void SetOccupationIndustry()
+    {
+      FetalDeathRecord fetalDeathRecord = new FetalDeathRecord();
+      fetalDeathRecord.MotherOccupation = "Carpenter";
+      Assert.Equal("Carpenter", fetalDeathRecord.MotherOccupation);
+      fetalDeathRecord.MotherIndustry = "Construction";
+      Assert.Equal("Construction", fetalDeathRecord.MotherIndustry);
+      fetalDeathRecord.FatherOccupation = "Lawyer";
+      Assert.Equal("Lawyer", fetalDeathRecord.FatherOccupation);
+      fetalDeathRecord.FatherIndustry = "Legal Services";
+      Assert.Equal("Legal Services", fetalDeathRecord.FatherIndustry);
+
+      //ije translations
+      IJEFetalDeath ije = new IJEFetalDeath(fetalDeathRecord);
```

Comment:
set the optional param to false to disable validation, new IJEFetalDeath(fetalDeathRecord, false)

Reply:
Allowing validation to occur, but deleting invalid FHIR fields from fixture